jsonmap_test.go

```diff
@@ -1708,3 +1711,141 @@ func TestValidThingWithEnumerableInterface(t *testing.T) {
 		require.EqualValues(t, validCase.Expected, dest)
 	}
 }
+
+type DogStruct struct {
```

Comment:
A boolean mapper would be useful (`true/false` literals)

Reply:
added